thirdparty/vulkan/vk_enum_string_helper.h

```diff
@@ -13,11 +13,7 @@
 #ifdef __cplusplus
 #include <string>
 #endif
-#ifdef USE_VOLK
-    #include <volk.h>
-#else
-    #include <vulkan/vulkan.h>
-#endif
+#include "drivers/vulkan/godot_vulkan.h"
```

Comment:
Changes in third party files should be marked by `/* GODOT start */`, `/* GODOT end */` comments and/or have patches included (usually in the `patches` subfolder), and mentioned in the `thirdparty/README.md` (this includes seems to be part of existing patches, which should be updated).

Reply:
Done.